guides/common/modules/proc_installing-and-configuring-puppet-agent-during-host-registration.adoc

```diff
@@ -46,11 +46,20 @@ endif::[]
 . In the {ProjectWebUI}, navigate to *Configure* > *Global Parameters* to add host parameters globally.
 Alternatively, you can navigate to *Configure* > *Host Groups* and edit or create a host group to add host parameters only to a host group.
 . Enable the Puppet agent using a host parameter in global parameters or a host group.
-ifdef::katello,orcharhino,satellite[]
+// consolidate with line 54 as soon as Satellite client contains Puppet agent 8
+ifdef::satellite[]
++
 Add a host parameter named `enable-puppet7`, select the *boolean* type, and set the value to `true`.
 endif::[]
-ifndef::katello,orcharhino,satellite[]
-Add a host parameter named `enable-official-puppet7-repo`, select the *boolean* type, and set the value to `true`.
+ifdef::katello,orcharhino[]
++
+* To use Puppet 8, add a host parameter named `enable-puppet8`, select the *boolean* type, and set the value to `true`.
+* To use Puppet 7, add a host parameter named `enable-puppet7`, select the *boolean* type, and set the value to `true`.
+endif::[]
+ifdef::foreman-el,foreman-deb[]
++
+To use Puppet 8, add a host parameter named `enable-official-puppet8-repo`, select the *boolean* type, and set the value to `true`.
```

Comment:
Just for completeness: this parameter does exist since https://github.com/theforeman/foreman/commit/c35092bdba5d15c2284d88f3de555f074a5773cf